### ISSUE TITLE
Install Finished landing page should have a link to login

### ISSUE THREAD

**Issue by greezybacon** [osTicket/osTicket-1.7#521]
After the installation is finished, the install-complete page should have a link to the scp login page.

**Comment by tomashastings** [comment]
You must have missed it, there /is/ a link to the scp-login page, it's on the right hand site, screenshot here: 

![osticketlink](https://f.cloud.github.com/assets/4083873/348737/0366ffc2-9f7d-11e2-8a49-88988176bb8f.jpg)

**Comment by greezybacon** [comment]
Yeah, there's also a handful at the bottom. This comes down to a design issue. There's too much crap on the screen. The only thing you see without five minutes of reading is a file permission correction, which most people probably ignore. 

From the design perspective, I (as an installer) think through: I'm done, now what? And this screen doesn't seem to answer that question well.

I think the red text and the arrow pointing out the what's next box is actually a good representation of the issue I'm raising.

**Comment by tomashastings** [comment]
You're right. There is way too much information in this screen. I wish I had more time, I'd like to fix a bunch of things I think is wrong with osTicket and make all the fixes/improvements available to everyone, there just aren't enough hours in a week for me to be able to do that though :(
